### ISSUE TITLE
WebApi call bug QueryConnection::DefaultChannel

### ISSUE THREAD

**Issue by Mareczin** [Splamy/TS3AudioBot#284]
Hi, I am migrating my system to the TS3AudioBot engine, but I have noticed a slight error probably. When I refer to such an api call:
`http://localhost:8180/api/settings/QueryConnection::DefaultChannel`

I receive this answer:
`{"ErrorCode":10,"ErrorName":"CommandError","ErrorMessage":"Found more than one matching key: \n QueryConnection::DefaultChannel\n QueryConnection::DefaultChannelPassword"}`

However, when I appeal to this second value:
`http://localhost:8180/api/settings/QueryConnection::DefaultChannelPassword`
Answer:
`{"Key":"QueryConnection::DefaultChannelPassword","Value":"3"}`

How can I refer to DefaultChannel when I get the answer that it is such a value, but it does not use it?
 would like to be able to refer to DefaultChannel and not to DefaultChannel Password in order to change.

**Comment by Flakebi** [comment]
You are right, this is a bug in the current system. It will be fixed once the new config system is ready.
The problem is that settings use substring matching, e.g. `QueryConnection::DefaultCh` should also match these settings.
Probably this can be fixed by changing the matcher to an `ExactFilter`.

https://github.com/Splamy/TS3AudioBot/blob/cb1448a5c30edf4649f8f49fa4c0212096764515/TS3AudioBot/MainCommands.cs#L1039

**Comment by Mareczin** [comment]
When new config will be ready? This feature in my opinion is very important. :/ 
Can not fix it for a while now?
